specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -14639,6 +14663,7 @@
     "ComputeNodeDeallocationOption": {
       "type": "string",
       "title": "Determines what to do with a node and its running task(s) after it has been selected for deallocation.",
+      "description": "Values are: requeue - Terminate running tasks and requeue them. The tasks will run again when the job is enabled. Remove nodes as soon as tasks have been terminated; terminate - Terminate running tasks. The tasks will not run again. Remove nodes as soon as tasks have been terminated; taskcompletion - Allow currently running tasks to complete. Schedule no new tasks while waiting. Remove nodes when all tasks have completed; retaineddata - Allow currently running tasks to complete, then wait for all task data retention periods to expire. Schedule no new tasks while waiting. Remove nodes when all task retention periods have expired. The default value is requeue.",
```

Comment:
nit: for better documentation, consider inserting newline characters here (like so https://github.com/Azure/azure-rest-api-specs/blob/current/specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/advisors.json#L54)